### ISSUE TITLE
R2: General comments

### ISSUE THREAD

**Issue by cliffckerr** [sciris/sciris#443]
General concerns

I am not convinced by the generality or ease of use claims that the paper makes. First, I have issue with the implied generality of sciris. There is a significant amount of scientific software that is not covered by the helper functions here. I agree with @aflaxman that the audience for this work needs to be better specified. Second, I the claims of "simple", "easy", "pleasent", and "dejargonifitation" are highly subjective (and I disagree with them in most cases!). Software seems simple when the assumptions in the code matches the assumptions in the user head, connecting back to the first concern I think identifying what domain (I assume epidemiology based on the examples of usage) this is aimed at would go a long way to ameliorating my concern. I also am not convinced by the claims that this will improve the long term maintainability of code. There are many more people who can read "plain" numpy / pandas code and you can google for numpy support. There is a trade off between short-term brevity and the long-term pool of people who can easily read and adapt the code.

I think the paper needs a discussion of the learning curve when users need to move beyond sciris, either because they need to do something it does not support yet or because they begin to need to make different assumption.

**Comment by cliffckerr** [comment]
cc https://github.com/openjournals/joss-reviews/issues/5076

**Comment by cliffckerr** [comment]
I appreciate these comments, and I definitely agree that whether or not something is "easy" is in the eye of the beholder! I can definitely relate -- I personally find pandas and seaborn confusing, and much prefer plain Numpy and Matplotlib, but I know many people disagree with me. 

I feel that seaborn is the most comparable library to Sciris, in that it extends/simplifies Matplotlib in a way that its author found useful, and many other people found useful too. I ended up quoting from the seaborn paper directly, because I think Michael put it more clearly than I could think how to: 
> Like seaborn, Sciris aims to "facilitate rapid exploration and prototyping through named functions and opinionated defaults" [@waskom2021seaborn].

I've tried to clarify who will, and won't, benefit most from Sciris:
> For some scientists, this increasing reliance on software has created a barrier between themselves and the science they want to do. It is these people – people who want things to "just work" rather than worry about the implementation details – who are the primary audience for Sciris. (In contrast, people who care a lot about implementation details – such as those who love using type hints – will likely *not* find Sciris to be as helpful.)

I also added a "Beyond Sciris" section at the end of the paper, including the specific example of parallelization as a case where Sciris' "opinions" might not go far enough. I hope these changes address these concerns, but of course happy to modify or add more detail.
